pylinac/core/profile.py

```diff
@@ -159,6 +160,31 @@ def values(self, value):
             raise TypeError("Values must be a numpy array")
         self._values = value.astype(float)
 
+    @property
+    def dpmm(self) -> NumberLike:
+        """The Dots-per-mm of the profile, defined at isocenter. E.g. if an EPID image is taken at 150cm SID,
+        the dpmm will scale back to 100cm. Added by ACC 3/12/2020"""
+        return self._dpmm
+
+    @dpmm.setter
+    def dpmm(self, value):
+        """The Dots-per-mm of the profile, defined at isocenter. E.g. if an EPID image is taken at 150cm SID,
+        the dpmm will scale back to 100cm. Added by ACC 3/12/2020"""
+        if value > 0:
+            self._dpmm = value
+
+    def profile_center(self) -> Tuple[NumberLike, NumberLike]:
```

Comment:
Users know this is a profile since the class has profile in the name. Perhaps a better name would be "center" or "center_data" or "midpoint".

Reply:
I have no problem renaming this to center. Will change and commit.